apis/python/src/tiledbsoma/_dask/README.md

```diff
@@ -0,0 +1,56 @@
+# `tiledbsoma._dask`
+Utilities for exporting [`Experiment`]s and [`ExperimentAxisQuery`]s to [anndata], with [Dask Array] `X` matrices.
+
+See also: [Tutorial: Scanpy + Dask-AnnData][tutorial]
+
+Passing a `tiledbsoma._dask.util.DaskConfig` to `Experiment.to_anndata` or `ExperimentAxisQuery.to_anndata` will cause the resulting [anndata]'s `X` matrix to be a [Dask Array].
```

Comment:
AFAIK, there is no such API as `Experiment.to_anndata` (intentionally). There is an `Experiment.axis_query`, which is a convenience method to create an `ExperimentAxisQuery` on that `Experiment`.

Reply:
Good catch, I meant `tiledbsoma.io.to_anndata`, fixed.